test/src/subnet/main_subnet.cpp

```diff
@@ -146,5 +150,14 @@ int main() {
     auto elapsed = std::chrono::duration_cast <std::chrono::duration<double, 
                    std::ratio < 1>>> (high_resolution_clock::now() - start).count();
     cout << "Simulation took:" << elapsed << "sec" << endl;
+
+    char sim_input[] = SIMULATOR_OUTPUT_PATH;
```

Comment:
see previous comments and make changes

Reply:
update line 90. create new #define statement

string input_data = "test/data/subnet_input_test.txt";